tasks/main.yml

```diff
@@ -14,7 +14,34 @@
     # it.
     state: latest  # noqa 403
     update_cache: yes
-  when: ansible_os_family == "Debian"
+  when: >
+    ansible_os_family == "Debian" and
+    ansible_distribution_release != "kali-rolling"
+
+- name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
+  apt:
+    name: 'cryptsetup-initramfs'
+    state: latest  # noqa 403
```

Comment:
You should add a comment above this line stating what this mysterious code does.  Same thing below where you use the magic incantation `noqa 303 305`.

Reply:
Addressed in commit 9f51229

Reply:
Why is "latest" necessary here if you're only installing a single package?  Seems it should work with `present` instead of `latest`.